### ISSUE TITLE
Metadata does not capture Constant queries

### ISSUE THREAD

**Issue by yisraelU** [disneystreaming/smithy4s#690]
The metadata schema visitor does not capture any constant queries params and therefore when rendering the metadata it can result in an unintended url with a different interpretation
```
url = "http://yoururl/hello?number=42"
```
when appending query parameter name=bob will result in
```
url = "http://yoururl/hello?number=42?name=bob"
```
with the last question mark parsed as part of the value for number

**Comment by daddykotex** [comment]
multiple `?` is valid? TIL

**Comment by Baccata** [comment]
Fixed since 0.17.2
